backend.py

```diff
@@ -279,6 +339,9 @@ def get_frame(self):
         # Overwrite with Capture timestamp
         frame.timestamp = self.g_pool.get_timestamp()
 
+        if self._enable_ir_camera:
+            frame._show_ir = True
```

Comment:
This implementation leads to the problem that depth and ir frames are mixed within a recorded video. For the Realsense backend, we have solved this issue by having to separate frames, a color and a depth frame. Both are recorded in separated videos. `Realsense_Source` has a custom `gl_display()` implementation that respects the user setting of what to display. I think something similar would be beneficial here as well.

Reply:
I pushed some changes that fix this issue.